### PR TITLE
MacOS support

### PR DESCRIPTION
**Pull request by AGulev**

### CODE REVIEW

File: filedrop/src/filedrop_macos.mm [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+@interface FileDropDelegate: NSView<NSDraggingDestination>
+@end
+
+FileDropDelegate *dragView;
+OnEventCallback draggingCallback;
+
+void FileDrop_PlatformSetEventListener(OnEventCallback callback) {
+    draggingCallback = callback;
+    if (dragView == 0)
```
Comment (AGulev) [inline filedrop/src/filedrop_macos.mm:15]: Unf. we can't set delegate for the main window, so it's easy to create a new invisible view with the same size window has and stretch it together with window.

File: filedrop/include/filedrop.h [hunk 2]
```diff
@@ -2,12 +2,13 @@
...
 #include <dmsdk/sdk.h>
 
-#if defined(DM_PLATFORM_HTML5)
+#if defined(DM_PLATFORM_HTML5) || defined(DM_PLATFORM_OSX)
 
 typedef void (*OnEventCallback)(const char* event, const char* filename, const uint8_t* data, int data_length);
 
 extern "C" {
     void FileDrop_PlatformSetEventListener(OnEventCallback callback);
+    void FileDrop_Finilize();
```
Comment (britzl) [inline filedrop/include/filedrop.h:11]: Typo: Finilize -> FinAlize

**Review by britzl — CHANGES_REQUESTED** [review]
One typo, but otherwise looking good!

File: filedrop/include/filedrop.h (hunk 2 above)
Comment (AGulev) [reply filedrop/include/filedrop.h]: Thx!

**Review by britzl — APPROVED** [review]

**Pull request merged**